source/index.ts

```diff
@@ -99,6 +99,20 @@ export default class PQueue<QueueType extends Queue<EnqueueOptionsType> = Priori
 		return this._pendingCount < this._concurrency;
 	}
 
+	get concurrency(): number {
+		return this._concurrency;
+	}
+
+	set concurrency(newConcurrency: number) {
+		if (!(typeof newConcurrency === 'number' && newConcurrency >= 1)) {
+			throw new TypeError(`Expected \`concurrency\` to be a number from 1 and up, got \`${newConcurrency}\` (${typeof newConcurrency})`);
+		}
```

Comment:
This validation is now defined twice. Once here and once in the constructor. Instead, you can just assign `concurrency` instead of `_concurrency` in the constructor.

https://github.com/sindresorhus/p-queue/pull/52/files#diff-21e9ddd93162651bd36f6e5bbfca8460R88

Reply:
Good idea, will do.